### PR TITLE
Table assign bijit

### PR DESCRIPTION
**Pull request by BijitMushahary**

### CODE REVIEW

File: app/code/local/Excellence/Test/controllers/IndexController.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?php
+class Excellence_Test_IndexController extends Mage_Core_Controller_Front_Action
+{
+    public function deleteAction()
+    { 
+        $this->loadLayout();            
+        $this->renderLayout();
```
Comment (mageexxcel) [inline app/code/local/Excellence/Test/controllers/IndexController.php]: this should always come in the end

File: app/design/frontend/default/models/template/test/head.phtml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
```
Comment (mageexxcel) [inline app/design/frontend/default/models/template/test/head.phtml]: this entire file is wrong.

File: app/code/local/Excellence/Test/controllers/IndexController.php [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+{
+    public function deleteAction()
+    {   
+        if( $this->getRequest()->getParam('user_id') > 0 ) {
+         try {
+              $user_model = Mage::getModel('user/user');
+              $address_model = Mage::getModel('user/address');
+              $user_model->setId($this->getRequest()->getParam('user_id'))->delete();
+              $address_model->setId($this->getRequest()->getParam('user_id'))->delete();
+              Mage::getSingleton('adminhtml/session')->addSuccess(Mage::helper('adminhtml')->__('Item was successfully deleted'));
```
Comment (mageexxcel) [inline app/code/local/Excellence/Test/controllers/IndexController.php]: does this message show? because u using admin session in frontend?

File: app/code/local/Excellence/Test/controllers/IndexController.php [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+class Excellence_User_IndexController extends Mage_Core_Controller_Front_Action
+{
+    public function deleteAction()
+    {   
+        if( $this->getRequest()->getParam('user_id') > 0 ) {
+         try {
+              $user_model = Mage::getModel('user/user');
+              $address_model = Mage::getModel('user/address');
+              $user_model->setId($this->getRequest()->getParam('user_id'))->delete();
+              $address_model->setId($this->getRequest()->getParam('user_id'))->delete();
```
Comment (mageexxcel) [inline app/code/local/Excellence/Test/controllers/IndexController.php]: does delete work? because user_id cannot be same for both user_model and address_model

File: app/code/local/Excellence/Test/controllers/IndexController.php [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+        }
+      }
+  public function saveAction() {
+    $fields = $this->getRequest()->getParams();
+    $user_model = Mage::getModel("user/user");
+    $address_model = Mage::getModel("user/address");
+    $user_model->setData($fields);
+    $address_model->setData($fields);
+    $user_model->save();
+    $address_model->save();
```
Comment (mageexxcel) [inline app/code/local/Excellence/Test/controllers/IndexController.php]: address_model needs to have a field called user_id to link user with address. without that its not possible to for which user has what addresss

File: app/design/frontend/default/models/layout/test.xml [hunk 6]
```diff
@@ -1,9 +1,19 @@
...
+  </reference>
+ </user_index_index>
+ <user_form_printform>
+    <reference name="content">
+            <block type="user/user" name="test" template="test/table.phtml"  />            
+     </reference>
+ </user_form_printform>
+ <default>
+    <reference name="head">
+      <action method="addItem"><type>skin_css</type><name>css/bootstrap.min.css</name></action>
```
Comment (mageexxcel) [inline app/design/frontend/default/models/layout/test.xml]: why is bootstrap used in this?

File: app/design/frontend/default/models/template/test/head.phtml [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+<meta name="keywords" content="<?php echo htmlspecialchars($this->getKeywords()) ?>" />
+<meta name="robots" content="<?php echo htmlspecialchars($this->getRobots()) ?>" />
+<link rel="icon" href="<?php echo $this->getFaviconFile(); ?>" type="image/x-icon" />
+<link rel="shortcut icon" href="<?php echo $this->getFaviconFile(); ?>" type="image/x-icon" />
+<!--[if lt IE 7]>
+<script type="text/javascript">
+//<![CDATA[
+    var BLANK_URL = '<?php echo $this->helper('core/js')->getJsUrl('blank.html') ?>';
+    var BLANK_IMG = '<?php echo $this->helper('core/js')->getJsUrl('spacer.gif') ?>';
+//]]>
```
Comment (mageexxcel) [inline app/design/frontend/default/models/template/test/head.phtml]: why is this file changed?

File: app/design/frontend/default/models/template/test/test.phtml [hunk 8]
```diff
@@ -1,188 +1,48 @@
...
-   }
- ?>
- </table>
- </div>
- <!--=============================================================================================-->
-
-</div>
- <script type="text/javascript">
-  $("input[type='text']").on("keyup", function()
+  $user_model = Mage::getModel('user/user')->getCollection();
```
Comment (mageexxcel) [inline app/design/frontend/default/models/template/test/test.phtml]: this database function should go in block class. never we shold write db code in phtml

File: app/code/local/Excellence/User/controllers/IndexController.php [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+      {
+       $address_user_id = $data->getUserid();
+       if ($address_user_id == $user_id) // used to identify common user id of user table and address table
+        {
+         $address_id = $data->getAddressId(); //passing address id to delete data from address table
+         $data->setId($address_id)->delete();
+        }
+      }
+     $this->_redirect('*/*/index');
+    }
```
Comment (sidexcel) [inline app/code/local/Excellence/User/controllers/IndexController.php]: put an else condition also

File: app/code/local/Excellence/User/controllers/IndexController.php [hunk 10]
```diff
@@ -0,0 +1,42 @@
+<?php
+class Excellence_User_IndexController extends Mage_Core_Controller_Front_Action
+{
+ public function deleteAction()
+  {   
+   if( $this->getRequest()->getParam('user_id') > 0 )
+    {
+     $user_id = $this->getRequest()->getParam('user_id');
```
Comment (sidexcel) [inline app/code/local/Excellence/User/controllers/IndexController.php]: create a function in model file called deleteUser and put all this code there. 

controller should always have as much less code as possible. this is very important, all keep as less code in controller as possoble.

File: app/code/local/Excellence/User/controllers/IndexController.php [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+        }
+      }
+     $this->_redirect('*/*/index');
+    }
+  }
+  public function saveAction() 
+   {
+    $fields = $this->getRequest()->getParams();
+    $user_model = Mage::getModel("user/user");
+    $address_model = Mage::getModel("user/address");
```
Comment (sidexcel) [inline app/code/local/Excellence/User/controllers/IndexController.php]: in user model add a function save user and move this code there

File: app/design/frontend/default/models/layout/user.xml [hunk 12]
```diff
@@ -0,0 +1,19 @@
...
+  <reference name="content">
+   <block type="user/user" name="user" template="user/user.phtml" />
+  </reference>
+ </user_index_index>
+ <user_form_printform>
+    <reference name="content">
+            <block type="user/user" name="user" template="user/table.phtml" /> 
+    </reference>
+ </user_form_printform>
+  <default>
```
Comment (sidexcel) [inline app/design/frontend/default/models/layout/user.xml]: remove bootstrap is not needed

File: app/design/frontend/default/models/template/user/table.phtml [hunk 13]
```diff
@@ -0,0 +1,54 @@
+<form role="form" action="<?php echo $this->getUrl('user/index/save'); ?>" method="post">
+    <ul class="form-list">
+      <li>
+           <label for="firstname" class="required">FIRST NAME :</label>
```
Comment (sidexcel) [inline app/design/frontend/default/models/template/user/table.phtml:4]: FIRST NAME dont write text like this. already commneted about this vihor pull request
